src/devices/Ahtxx/AhtBase.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+using System.Threading;
+using UnitsNet;
+
+namespace Iot.Device.Ahtxx
+{
+    /// <summary>
+    /// AHT temperature and humidity sensor family.
+    /// Note: has been tested with AHT20 only, but should work with AHT10 and AHT15 as well.
+    /// Up to now all functions are contained in the base class, though there might be differences between the sensors types.
+    /// </summary>
+    public class AhtBase : IDisposable
+    {
+        /// <summary>
+        /// Address of AHTxx device (0x38). This address is fix and cannot be changed.
+        /// This implies that only one device can be attached to a single I2C bus at a time.
+        /// </summary>
+        public const int DeviceAddress = 0x38;
```

Comment:
we call this constant DefaultI2cAddress everywhere

Reply:
Changed. But I think this name is misleading, as the device supports exactly one address and 'default' suggests that there are more options available.